pubspec.yaml

```diff
@@ -43,3 +43,11 @@ dev_dependencies:
   test: ^1.3.4
   tuneup: any
   yaml: any
+dependency_overrides:
+  # remove once https://github.com/flutter/devtools/pull/1019 is released
```

Comment:
It might be worth asking if you can become an uploader for some of the packages you're contributing to, as a way of saving this type of workaround.

Reply:
good idea, done.